src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -158,20 +158,143 @@ For languages that New Relic does not currently provide an agent or SDK, OpenTel
+In general, New Relic's APM agents will collect more telemetry data for your services, and they offer a wide range configuration options and an extensive set of auto-instrumentation capabilities.
 
-APM agents offer detailed transaction trace visibility for individual services. They also offer predefined sampling to balance the performance impact of your instrumentation against the need to capture enough data to gain helpful insights.
+New Relic's APM agents offer detailed transaction trace visibility for individual services. They also offer predefined sampling to balance the performance impact of your instrumentation against the need to capture enough data to gain helpful insights.
 
 ## How OpenTelemetry works with New Relic [#how-it-works]
 
-New Relic supports the native OpenTelemetry Protocol (OTLP) for exporting telemetry data. This allows you to use the vendor neutral components developed by the OpenTelemtry community to export your data to New Relic.
+New Relic supports the native OpenTelemetry Protocol (OTLP) for exporting telemetry data. This allows you to use the vendor neutral components developed by the OpenTelemetry community to export your data to New Relic.
+
+The following tables show the supported features for each telemetry signal. If you have questions about these or have an unsupported use case, please contact us in our [CNCF Slack channel](https://cloud-native.slack.com/archives/C024DRQ63UP).
 
 <Callout variant="important">
 [New Relic's exporters](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-legacy-new-relic-exporters) for OpenTelemetry are now deprecated in favor of exporting data to New Relic using OTLP.
 </Callout>
 
+### Traces [#trace-ingest]
+
+New Relic offers support for the OTLP ingest of trace signals. The maturity of the upstream specification is stable. 
+
+OpenTelemetry traces and spans are compatible with New Relic traces and spans. OpenTelemetry spans optionally include attributes (name-value pairs) and resource attributes which map directly to dimensions that can be used to facet or filter span data at query time. OpenTelemetry span metadata (for example, `name`, `kind`, and `trace_id`) also map directly to dimensions on NewRelic spans. At this time, New Relic does not support span links or array attributes.
+
+For details, see the Traces section of our [best practices](content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx) guide.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>Feature</th>
+      <th>Supported</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>[Span events](https://github.com/open-telemetry/opentelemetry-specification/blob/f4fd01256675ab12ecf2abf6aa8525f9a14a9ec1/specification/trace/api.md#add-events)</td>
```

Comment:
@mmfred, I wish we could control the width of tables, but alas, I don't think we can. Originally, these tables had description columns, but we took them out to avoid redundancy with another topic. Let's address this later with a CSS fix.